cmd/metal-api/internal/service/machine-service.go

```diff
@@ -328,6 +328,16 @@ func (r machineResource) webService() *restful.WebService {
 		Returns(http.StatusOK, "OK", v1.MachineRecentProvisioningEvents{}).
 		DefaultReturns("Error", httperrors.HTTPErrorResponse{}))
 
+	ws.Route(ws.POST("/events").
+		To(editor(r.addProvisioningEvents)).
+		Operation("addProvisioningEvents").
+		Doc("adds machine provisioning events").
+		Metadata(restfulspec.KeyOpenAPITags, tags).
+		Reads(v1.MachineProvisioningEvents{}).
+		Writes(v1.MachineRecentProvisioningEventsMap{}).
+		Returns(http.StatusOK, "OK", v1.MachineRecentProvisioningEventsMap{}).
```

Comment:
If there are many, is it really worth it to send them back?

Reply:
Changed to a proper response